detekt-core/src/main/resources/default-detekt-config.yml

```
@@ -363,6 +363,10 @@ formatting:
 
 naming:
   active: true
+  BooleanPropertyNaming:
+    active: false
+    allowedPrefixes: '^(is|has|should|need|noNeed|was|are|may|can|had|for|with)'
```

```suggestion
    allowedPrefixes: '^(is|has|are)'
```

I believe the default value is a bit too long? 🤔 Or at least is really opinionated. I would opt for a shorter list if possible.

---

maybe  '^(is|has|are|should)' ? 🤔

but the current long list is not bad either